eccehomo/app.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from flask import Flask, abort, request, send_from_directory, jsonify
+from webargs.flaskparser import use_kwargs
+
+from .fields import image_args, correct_arguments
+from .settings import MEDIA_ROOT, MEDIA_URL
+from .utils import create_image, get_new_filename
+
+app = Flask(__name__)
+
+
+@app.route(os.path.join('/', MEDIA_URL, "<path:filename>"))
+@use_kwargs(image_args, validate=correct_arguments)
+def get_image(filename, **kwargs):
+    absolute_path = os.path.join(MEDIA_ROOT, filename)
+    if not os.path.isfile(absolute_path):
+        abort(404)
+
+    resized_filename = get_new_filename(filename, request)
+    resized_absolute_path = os.path.join(MEDIA_ROOT, resized_filename)
+
+    if os.path.isfile(resized_absolute_path):
+        return send_from_directory(MEDIA_ROOT, resized_filename)
+
+    if create_image(absolute_path, resized_absolute_path, **kwargs):
```

Comment:
Making a function to return a boolean to check if smth went OK is kindy tricky, isn't it?
![64347038](https://cloud.githubusercontent.com/assets/1484589/20721738/2aee9594-b664-11e6-854b-eaf210de0acd.jpg)